config/routes.rb

```diff
@@ -87,6 +87,7 @@
   get '/import/download_guidance', to: 'import#download_guidance'
 
   get '/patients/:id/household_removeable', to: 'patients#household_removeable'
+  get '/patients/:id/monitoree_unavailable', to: 'patients#monitoree_unavailable'
```

Comment:
IMO this page should be at`/monitoree_unavailable`  rather than being a child of `/patients/:id`. It just feels wrong to me to make it a child page of a resource that might not actually exist.

Reply:
I agree with this, changed here: https://github.com/SaraAlert/SaraAlert/pull/736/commits/9e042dabe51531ab4bb5f4bbe2e37aea138db260